### DISCUSSION TITLE
Welcome to galoy Discussions!

### DISCUSSION THREAD

**Discussion by bodymindarts** [GaloyMoney/galoy#1265] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nicolasburtey** [comment]
hey @bodymindarts, new interesting features from github. let's get the discussions going!

**Comment by zoopli** [comment]
Hi all,
I'm zoop and planning on running Galoy in  baremetal/vps environment outside of google cloud.
It won't be tied to a specific location like most projects but serve the needs of an online-community.
As it's my first time contributing to an OSS project I hope to be of help in modifying the charts in that direction.
